sdk/communication/azure-communication-callautomation/src/main/java/com/azure/communication/callautomation/CallMediaAsync.java

```diff
@@ -210,6 +214,32 @@ Mono<Response<Void>> recognizeWithResponseInternal(CallMediaRecognizeOptions rec
                     .setRecognizeOptions(recognizeOptionsInternal)
                     .setOperationContext(recognizeOptions.getOperationContext());
 
+                return contentsInternal.recognizeWithResponseAsync(callConnectionId, recognizeRequest, context);
+            } else if (recognizeOptions instanceof CallMediaRecognizeSpeechOptions) {
+                CallMediaRecognizeSpeechOptions speechRecognizeOptions = (CallMediaRecognizeSpeechOptions) recognizeOptions;
+
+                SpeechOptionsInternal speechOptionsInternal = new SpeechOptionsInternal().setEndSilenceTimeoutInMs(speechRecognizeOptions.getEndSilenceTimeoutInMs());
```

Comment:
is this the only attribute SpeechOptionsInternal has?

Reply:
yes, SpeechOptionsInternal  is autogenerated